tests/store/artifact/test_models_artifact_repo.py

```diff
@@ -133,13 +135,15 @@ def test_models_artifact_repo_add_registered_model_meta_file():
     artifact_path = "artifact_path"
     dst_path = "dst_path"
     artifact_location = f"s3://blah_bucket/{artifact_path}"
-    artifact_dst_path = f"{dst_path}/{artifact_path}"
+    artifact_dst_path = f"{dst_path}/{artifact_path}/"
     model_name = "MyModel"
     model_version = "12"
 
     with mock.patch.object(
         MlflowClient, "get_model_version_download_uri", return_value=artifact_location
-    ), mock.patch("mlflow.store.artifact.models_artifact_repo.write_yaml") as write_yaml_mock:
+    ), mock.patch("os.path.isdir", return_value=True), mock.patch(
```

Comment:
Do we really need to mock `os.path.isdir`?

Reply:
refactored tests to use tmp_path location